Spanner/src/Operation.php

```diff
@@ -710,6 +710,14 @@ private function flattenKeySet(KeySet $keySet)
         return $this->arrayFilterRemoveNull($keys);
     }
 
+    private function getFullyQualifiedDatabaseName($session)
```

Comment:
Perhaps `getDatabaseNameFromSession`? Also, be sure to type-hint the argument:

```suggestion
    private function getDatabaseNameFromSession(Session $session)
```